solr/core/src/java/org/apache/solr/servlet/PrometheusMetricsServlet.java

```diff
@@ -69,7 +69,8 @@ public final class PrometheusMetricsServlet extends BaseSolrServlet {
       new OsMetricsApiCaller(),
       new ThreadMetricsApiCaller(),
       new StatusCodeMetricsApiCaller(),
-      new CoresMetricsApiCaller()
+      new CoresMetricsApiCaller(),
```

Comment:
should we add `coreEnable` flag here to call one/other?

Reply:
Sure. Looks good to me!

Just to add, it seems solr.xml can process java system property we can use that to control or try on one node

```
<metrics enabled="${metricsEnabled:true}"/>
```